src/components/common/BookCard/BookCard.js

```diff
@@ -14,29 +14,29 @@ const BookCard = ({ children, book }) => {
           {children || (
             <>
               <div className="text-black">
-                <Typography variant="h6" className="BookCard__title">
+                <Typography variant="h6" className="BookCard__title text-10">
                   {book.title}
                 </Typography>
                 <div className="d-flex">
                   {book.categories.map((item, idx) => (
```

Comment:
I think it's better to avoid giving key as index as much as possible. And the category db may contain a unique key that can be used.

Reply:
For now, categories don't have separate ids. Hence, using keys